mscore/musescore.cpp

```diff
@@ -246,6 +252,9 @@ const std::list<const char*> MuseScore::_basicNoteInputMenuEntries {
             "",
             "pad-rest",
             "",
+            "pad-raise",
+            "pad-lower",
+            "",
```

Comment:
If anything MuseScore is overloaded compared to Finale, even without these new buttons.

(Finale doesn't show the double dot, or the double sharp, or the double flat, or the rest button, or the flip stem button. I think we could remove the double dot, double sharp, and double flat from the basic controls. But that's not really related to this pull request.)